### DISCUSSION TITLE
Adding specific list properties to slots not working

### DISCUSSION THREAD

**Discussion by MBlondin** [microsoft-search/pnp-modern-search#3563] [General]
**Version used**
4.10.2

**Describe the bug**
We deployed PnP Modern Search and on a page to search within a specific list.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/1737991/774141c2-f69e-40a8-b7ab-ad33f8e05aea)

We added the properties (list columns) 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/1737991/cdceb70d-abf7-4286-9736-e00cfb42703b)
 and added the custom slots
![image](https://github.com/microsoft-search/pnp-modern-search/assets/1737991/56adeb05-6239-4127-b1ad-4b534e0f7314)

The only property/Column that shows/works is Location.

**Comment by kasperbolarsen** [comment]
Where did you add the managed properties, in "Selected properties", I hope? 
Which layout are you using?
Adding the properties in the layout slots is just the first step, you will also have to add them to "Manage columns", depending to which layout you are using

**Comment by MBlondin** [comment]
Hi @kasperbolarsen I added tthe list properties in the raw properties
Using Handlebars/Html layout Cards
Managed columns?  do you mean
![image](https://github.com/microsoft-search/pnp-modern-search/assets/1737991/96e3a964-bd6b-4134-93b8-5ce42a6fca46)

**Comment by kasperbolarsen** [comment]
I would recommend looking at the managed properties first, using either the debug layout or one of the tools here, https://microsoft-search.github.io/pnp-modern-search/using-query-tools-to-verify-issues/

Once the managed properties contains the data you expect, you should add them to the "Selected properties" otherwise they are not available for any of the web parts/layouts
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/de57973f-7a44-4746-abd9-2249cdb5d0d8)
